erts/emulator/utils/make_version

```diff
@@ -52,6 +52,16 @@ my $architecture = shift;
 defined $architecture or die "No architecture specified";
 $architecture =~ s&^.*[/\\]&&;	# Remove directory part if any
 
+if ($architecture eq "win32") {
+  if (%ENV{"CONFIG_SUBTYPE"} eq "arm64" || %ENV{"CONFIG_SUBTYPE"} eq "x64_arm64") {
```

Comment:
This fails on old perl version. I think it should be '$ENV' instead of '%ENV'. What do you think @bjorng ?